packages/main/src/DayPicker.js

```diff
@@ -182,8 +182,10 @@ class DayPicker extends CalendarPart {
 
 		this._weeks = [];
 
+		const _isSecondaryCalendarType = this.hesSecondaryCalendarType;
```

Comment:
This variable is not needed. Just use the getter where needed

Reply:
Done